js/apps/account-ui/src/oid4vci/Oid4Vci.tsx

```diff
@@ -0,0 +1,218 @@
+import {
+    Select,
+    SelectList,
+    SelectOption,
+    PageSectionVariants,
+    PageSection,
+    ActionList,
+    ActionListItem,
+    List,
+    ListItem,
+    MenuToggleElement,
+    MenuToggle
+} from '@patternfly/react-core';
+import { useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { useAlerts } from "@keycloak/keycloak-ui-shared";
+import { usePromise } from "../utils/usePromise";
+import { Page } from "../components/page/Page";
+import { parseResponse } from "../api/parse-response";
+import { useEnvironment } from '../root/KeycloakContext';
+
+interface CredentialsIssuer {
+  credential_issuer: string;
+  credential_endpoint: string;
+  authorization_servers: string[]; 
+  credential_configurations_supported: Map<string, SupportedCredentialConfiguration>
+}
+
+interface SupportedCredentialConfiguration {
+  id: string,
+  format: string,
+  scope: string
+}
+
+type VCState = {
+  dropdownItems: string[],
+  selectOptions: Map<string, SupportedCredentialConfiguration>,
+  credentialIssuer?: CredentialsIssuer,
+  issuerDid: string,
+  qrCode: string,
+  isOpen: boolean,
+  offerQRVisible: boolean
+}
+
+export const Oid4Vci = () => {
```

Comment:
You must add this `Oid4Vci` to the `index.ts` so that people can reuse this screen

Reply:
Not sure if I missunderstand something, but I think I did: https://github.com/keycloak/keycloak/pull/29174/files#diff-2675d62d32855ae556405839057d5cbc7267f07c0a60f14b2e90227c9e5f02a7R42